launch/tier4_localization_launch/launch/twist_estimator/twist_estimator.launch.xml

```diff
@@ -1,10 +1,19 @@
 <?xml version="1.0"?>
 <launch>
-  <group>
+<group if ="$(eval &quot;'$(var twist_estimator_mode)'=='gyro_odom_fusion'&quot;)">
     <include file="$(find-pkg-share gyro_odometer)/launch/gyro_odometer.launch.xml">
-      <arg name="input_vehicle_twist_with_covariance_topic" value="/sensing/vehicle_velocity_converter/twist_with_covariance"/>
-      <arg name="output_twist_with_covariance_topic" value="/localization/twist_estimator/twist_with_covariance"/>
-      <arg name="output_twist_with_covariance_raw_topic" value="/localization/twist_estimator/twist_with_covariance_raw"/>
+        <arg name="input_vehicle_twist_with_covariance_topic" value="/sensing/vehicle_velocity_converter/twist_with_covariance"/>
+        <arg name="output_twist_with_covariance_topic" value="/localization/twist_estimator/twist_with_covariance"/>
+        <arg name="output_twist_with_covariance_raw_topic" value="/localization/twist_estimator/twist_with_covariance_raw"/>
     </include>
+</group>
+
+  <group if="$(eval &quot;'$(var twist_estimator_mode)'=='gyro_odom_gnss_fusion'&quot;)">
```

Comment:
As this conditional branch may appear a bit confusing, can you describe somewhere to clarify this design?
- e.g.) Add a comment here
- e.g.) Add something like a table ([maybe like this](https://github.com/autowarefoundation/autoware.universe/pull/2848#issuecomment-1461310294)) in readme.

Reply:
Added description to README.md of map4_localization_launch.